assets/js/lib/test-utils/factories/executions.js

```diff
@@ -17,15 +17,30 @@ export const executionValueFactory = Factory.define(({ sequence }) => ({
 export const executionExpectationEvaluationFactory = Factory.define(
   ({ sequence, params }) => {
     const name = params.name || `expectation_${sequence}`;
+    const type = params.type || expectationReturnTypeEnum();
+    const return_value = params.return_value || faker.datatype.number();
+    const failure_message = params.failure_message
+      ? { failure_message: params.failure_message }
+      : {};
 
     return {
       name,
-      return_value: faker.datatype.number(),
-      type: expectationReturnTypeEnum(),
+      return_value,
+      type,
+      ...failure_message,
     };
   }
 );
 
+export const failingExpectEvaluationFactory = Factory.define(({ params }) =>
+  executionExpectationEvaluationFactory.build({
+    ...params,
+    return_value: false,
+    type: 'expect',
+    failure_message: params.failure_message || faker.lorem.sentence(),
```

Comment:
I think no need for this:
```
failure_message: faker.lorem.sentence(),
```
would be enough and it would take the `faker` value only as default.
I think `fishery` is much more smarter than what we think hehe